docs/api-reference/certificates/v1alpha1/definitions.html

```diff
@@ -0,0 +1,1318 @@
+<!DOCTYPE html>
```

Comment:
Sorry about that. :( 
Deleted the file. Still do not understand how it got created though.